runtime/doc/debugger.txt

```diff
@@ -119,7 +119,6 @@ The following compile time preprocessor variables control the features:
     Debug Glyphs				FEAT_SIGNS
     Debug Highlights				FEAT_SIGNS
     Message Footer				FEAT_FOOTER
-    Balloon Evaluation				FEAT_BEVAL
```

Comment:
And if you do decide to do it here, make sure to remove section 3 as well: it's empty.